container/rkt/helpers.go

```diff
@@ -20,17 +20,56 @@ import (
 	"path"
 	"strings"
 
+	rktapi "github.com/coreos/rkt/api/v1alpha"
 	"github.com/golang/glog"
+	"golang.org/x/net/context"
 )
 
 type parsedName struct {
 	Pod       string
 	Container string
 }
 
-func verifyName(name string) (bool, error) {
-	_, err := parseName(name)
-	return err == nil, err
+func verifyPod(name string) (bool, bool, error) {
+	pod, err := cgroupToPod(name)
+
+	if err != nil || pod == nil {
```

Comment:
pretty sure you can, or at least I thought it as such (i.e. err would be for api returning an error, while pod == nil means api didn't return an error, but api couldn't match cgroup to pod)